### PR TITLE
Time picker interval

### PR DESCRIPTION
**Pull request by enchev**

### CODE REVIEW

File: apps/tests/ui/time-picker/time-picker-tests.ts [hunk 1]
```diff
@@ -58,6 +58,15 @@ if (platform.device.os === platform.platformNames.ios) {
     }
 }
 
+export function test_WhenCreated_MinuteIntervalIs1() {
+    helper.buildUIAndRunTest(_createTimePicker(), function (views: Array<viewModule.View>) {
+        var timePicker = <timePickerModule.TimePicker>views[0];
+        var actualValue = timePicker.minuteInterval;
+        var expectedValue = 1;
+        TKUnit.assert(actualValue === expectedValue, "Actual: " + actualValue + "; Expected: " + expectedValue);
```
Comment (hshristov) [inline apps/tests/ui/time-picker/time-picker-tests.ts]: Please use assertEqual for better error reporting when test fails.

File: ui/time-picker/time-picker.android.ts [hunk 2]
```diff
@@ -78,4 +78,8 @@ export class TimePicker extends common.TimePicker {
     public _setNativeTime() {
         this._setNativeValueSilently(this.hour, this.minute);
     }
+
+    public _setNativeMinuteIntervalTime() {
+        //
```
Comment (hshristov) [inline ui/time-picker/time-picker.android.ts]: Why there is no code here?

**Comment by enchev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by ErjanGavalji** [comment]
:+1:

**Comment by nsndeck** [comment]
:+1:

**Pull request merged**